users/models/hashtag_follow.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from django.db import models
+
+
+class HashtagFollowQuerySet(models.QuerySet):
+    def by_hashtags(self, hashtags: list[str]):
+        query = self.filter(hashtag_id__in=hashtags)
+        return query
+
+
+class HashtagFollowManager(models.Manager):
+    def get_queryset(self):
+        return HashtagFollowQuerySet(self.model, using=self._db)
+
+    def by_hashtags(self, hashtags: list[str]):
+        return self.get_queryset().by_hashtags(hashtags)
+
+
+class HashtagFollow(models.Model):
+    identity = models.ForeignKey(
+        "users.Identity",
+        on_delete=models.CASCADE,
+        related_name="hashtag_follows",
+    )
+    hashtag = models.ForeignKey(
```

Comment:
This is the target of that query in aget_targets, so I think it should have `db_index=True`

Reply:
Added 👍